conbench/templates/app.html

```diff
@@ -212,12 +211,12 @@ <h4 class="modal-title"><i class="bi bi-trash3-fill"></i></h4>
     Based on the indexes of the columns, input boxes are created. The search is triggered on
     key press.
 INPUTS:
- data_table_id: string
+ data_table_elem: DOM element for the <table>
  column_indexes: array
  */
-function free_text_column_search(data_table_id, column_indexes) {
+function free_text_column_search(data_table_elem, column_indexes) {
```

Comment:
CC @sayantikabanik by passing in a DOM element instead of a CSS selector string we can use these functions more flexibly (when a CSS selector string resolves to _multiple_ DOM elements).